source/connect/connection-options.txt

```diff
@@ -91,9 +91,9 @@ string, see :manual:`Connection Strings </reference/connection-string/>` in the
    * - **connectTimeoutMS**
      - non-negative integer
      - ``30000``
-     - Specifies the amount of time, in milliseconds, to wait to establish a single TCP
+     - Specifies the amount of time in milliseconds to wait to establish a single TCP
        socket connection to the server before raising an error. Specifying
-       ``0`` disables the connection timeout.
+       ``0``  means your application will use the operating system's default socket timeout value.
```

Comment:
This isn't accurate - a connectTimeoutMS of 0 sets an infinite socket timeout when establishing a socket.